3.preprocessing_features/2.bulk_processing.ipynb

```diff
@@ -241,11 +268,17 @@
     "    )\n",
     "\n",
     "    # Step 4: Normalization\n",
+    "    samples = (\n",
+    "        \"Metadata_cell_line == 'U2-OS'\" if normalize_with_U2OS else \"all\"\n",
+    "    )  # \"all\" is the default to perform on whole plate\n",
+    "    print(f\"Normalizing using samples: {samples}\")\n",
+    "\n",
```

Comment:
all would include U2-0S? I think I'm a little confused on if U2-0S is just being run as one sample or if all of the samples are being run and normalized to U2-0S

Reply:
The difference between all versus U2OS is that all will just normalize using all of the cells together across cell lines, while U2OS will generate the normalization function using just the U2OS cells and then apply to the whole plate. If it helps, here is the exact pycytominer documentation on the parameter:

samples (str) – The metadata column values to use as a normalization reference. We often use control samples. The function uses a pd.query() function, so you should structure samples in this fashion. An example is “Metadata_treatment == ‘control’” (include all quotes). Defaults to “all”.